README.md

```diff
@@ -238,12 +243,17 @@ objectives:
 - **targetTimeSlices** *numeric* *\[0.0, 1.0\]*, required only when budgeting
   method is set to TimeSlices
 
-- **indicator.ratioMetric** *Metric {Good, Total}*, if `ratioMetric` is defined
-    then `thresholdMetric` should not be set in `indicator`
+- **indicator.ratioMetric** *Metric {Good, Total} or {Bad, Total}*
+    if `ratioMetric` is defined then `thresholdMetric` should not be set in `indicator`
 
   - *Good* represents the query used for gathering data from metric sources used
    as the numerator. Received data is used to compare objectives (threshold)
-   values to find good values.
+   values to find good values. If `Bad` is defined then `Good` should not be set.
+
+  - *Bad* represents the query used for gathering data from metric sources used
```

Comment:
How would the math on this work?  Example being, to calculate reliability in a threshold metric we do `good/total`.  Should we infer `good` by subtracting `bad` from `total`?  Eg `(total-bad)/total`

Reply:
Love to hear more from @geototti21. Maybe I misunderstood the ticket. For me, the idea behind also allowing `Bad` option as for some people it makes more sense to think about error events instead of good events and based SLIs around that.

Reply:
Yeah, having `bad` events makes sense.  I'm curious about the math to invert it, since we should call it out in the spec.  Mainly because we want to have an apples to apples comparison when looking at reliability.  Example:

Sevice A has 99 good requests out of 100 total requests.  This could be said to be:
`99 (good) / 100 (total) = 0.99` = 99% reliable.

Service B has 1 error out of 100 total requests.  So that would be:
`1 (bad) / 100 (total) = 0.01`  which if we convert to a percentage = `1%`

For Service B, we wouldn't want to put an SLO on that, because for SLOs we don't want the number to fall below a given number, and in this scenario, we actually do want the number to go down, so our logic is reversed.  To fix this we can either subtract `bad` from `total` to get a derived `good` value, or we can subtract the quotient from `1` and use the difference as the numerator.  Ex (for service b):

`(100 - 1)/100 = 0.99 = 99%`
or
```
1/100 = 0.01 
1 - 0.01 = 0.99 = 99%
```

The first one makes more sense to me because it keep the logic closer to the `bad` section

Reply:
Thanks for picking this up @weyert! My thoughts on this (and how we already use this internally at my corp) are as @ian-bartholomew describes, for SLOs we can support  `(total-bad)/total` or `good/total` and then the rest of maths are the same.

So my suggestion was based on these types of use-case! The PR LGTM and covers #45 ✅ 😄

Reply:
Thanks @geototti21 ! 

@weyert Could you just add some language here describing the math? (the `(total-bad)/total` part)  Thanks!

Reply:
Sure, I will make sure its up-to-date before tomorrow's meeting :)

Reply:
I have updated the PR